src/com/lilithsthrone/game/character/body/types/EyeType.java

```diff
@@ -95,65 +302,14 @@ public String getNamePlural(GameCharacter gc) {
 
 	@Override
 	public String getDescriptor(GameCharacter gc) {
-		switch(this){
-			case ANGEL:
-				return UtilText.returnStringAtRandom("angelic");
-			case CAT_MORPH:
-				return UtilText.returnStringAtRandom("cat-like");
-			case COW_MORPH:
-				return UtilText.returnStringAtRandom("cow-like");
-			case DEMON_COMMON:
-				return UtilText.returnStringAtRandom("demonic");
-			case DOG_MORPH:
-				return UtilText.returnStringAtRandom("dog-like");
-			case SQUIRREL_MORPH:
-				return UtilText.returnStringAtRandom("squirrel-like");
-			case ALLIGATOR_MORPH:
-				return UtilText.returnStringAtRandom("reptile-like");
-			case HARPY:
-				return UtilText.returnStringAtRandom("bird-like");
-			case HORSE_MORPH:
-				return UtilText.returnStringAtRandom("horse-like");
-			case REINDEER_MORPH:
-				return UtilText.returnStringAtRandom("reindeer-like");
-			case HUMAN:
-				return UtilText.returnStringAtRandom("");
-			case LYCAN:
-				return UtilText.returnStringAtRandom("wolf-like");
-			case FOX_MORPH:
-				return UtilText.returnStringAtRandom("fox-like");
-			case RAT_MORPH:
-				return UtilText.returnStringAtRandom("rat-like");
-			case RABBIT_MORPH:
-				return UtilText.returnStringAtRandom("rabbit-like");
-			case BAT_MORPH:
-				return UtilText.returnStringAtRandom("bat-like");
-		}
-		return "";
+		return UtilText.returnStringAtRandom(StringUtils.split(this.descriptor, ','));
```

Comment:
You can use this to avoid the dependency:
```java
this.descriptor.split(",")
```
Or, if you feel that you need null-safety (which might still break returnStringAtRandom):
```java
this.descriptor != null ? this.descriptor.split(",") : null
```

Reply:
Right, in this instant I know the dependency doesn't buy me much. It is kind of habit for me to bring in this dependency, as most projects I've worked on have included it. Just kinda using it to figure out the dependency situation, since I was surprised to see none.

Reply:
I tend to look at this from the opposite angle, in that I avoid dependencies unless it provides a significant benefit, whether it be efficiency (number crunching) or correctness (crypto) or simplicity (lombok). To me, more dependencies means more to maintain, to update versions or make an artifactory or whatnot. I may be getting ahead of things a bit, but I don't think this really warrants the bother.

However, this isn't my call to make, it's Innoxia's. If she's cool with it, so am I.

Reply:
I certainly won't disagree with you that too many dependencies are a bad thing and that they should be vetted. I've spent many an hour in dependency hell and know it is no fun. Happy to make the change if that is what is requested.